components/02-elements/form/input/input.config.js

```diff
@@ -18,7 +18,10 @@ module.exports = {
     {
       name: 'inline',
       context: {
-        class: 'input--inline'
+        class: 'input--inline',
+        label: {
+          class: 'label--inline'
```

Comment:
We can extend `label--inline` styles for `input--inline`, will be easier than adding an additional class.

Reply:
I tried to follow BEM and add modifier where it should be. YOu mean by adding
```
.input--inline {
    .label {
        ....
    }
}
```